docs/source/serialization.rst

```diff
@@ -193,3 +193,56 @@ Specify relations by intid:
     "relatedItems": [347127075, 347127076]
   }
 
+Walking the content
+-------------------
+
+The serialized content have some hints on how to move to the items around.
+
+Parent
+^^^^^^
+
+The parent is the folder where the page is inserted.
+
+.. code-block:: json
+
+  {
+    "parent": {
+      "@id": "http://nohost/plone/folder-with-items", 
+      "@type": "Folder", 
+      "title": "Folder with items",
+      "description": "This is a folder with two documents", 
+    }
+  }
+
+Previous Item
+^^^^^^^^^^^^^
+
+The previous item is the sibling that comes before in the current folder.
```

Comment:
The "previous_item" attribute points to the sibling that is located before the current element in the parent container (Plone uses the getObjectPositionInParent attribute to sort content objects within a folderish container).